docs/source/qvm.rst

```diff
@@ -87,6 +95,75 @@ and then execute it with the QVM directly from the command line:
     [2018-11-30 11:13:58] Classical memory (low -> high indexes):
     [2018-11-30 11:13:58]     ro:  1 0
 
+The QVM offers a simple benchmarking mode with ``qvm --verbose
+--benchmark``. Example output looks like this:
+
+.. code::
+   $ ./qvm --verbose --benchmark
+   ******************************
+   * Welcome to the Rigetti QVM *
+   ******************************
+   Copyright (c) 2016-2019 Rigetti Computing.
+
+   (Configured with 8192 MiB of workspace and 8 workers.)
+
+   <135>1 2019-05-01T18:26:14Z workstation.local qvm 96177 - - Selected simulation method: pure-state
+   <135>1 2019-05-01T18:26:15Z workstation.local qvm 96177 - - Computing baseline serial norm timing...
+   <135>1 2019-05-01T18:26:15Z workstation.local qvm 96177 - - Baseline serial norm timing: 96 ms
+   <135>1 2019-05-01T18:26:15Z workstation.local qvm 96177 - - Starting "bell" benchmark with 26 qubits...
+
+   ; Transition H 0 took 686 ms (gc: 0 ms; alloc: 65536 bytes)
+   ; Transition CNOT 0 1 took 651 ms (gc: 0 ms; alloc: 0 bytes)
+   ; Transition CNOT 1 2 took 658 ms (gc: 0 ms; alloc: 32656 bytes)
+   ; Transition CNOT 2 3 took 661 ms (gc: 0 ms; alloc: 0 bytes)
+   ; Transition CNOT 3 4 took 650 ms (gc: 0 ms; alloc: 0 bytes)
+   ; Transition CNOT 4 5 took 662 ms (gc: 0 ms; alloc: 0 bytes)
+   ; Transition CNOT 5 6 took 673 ms (gc: 0 ms; alloc: 0 bytes)
+   [...]
+   <135>1 2019-05-01T18:30:13Z workstation.local qvm 96288 - - Total time for program run: 24385 ms
+
+The QVM also has mode for faster execution of long quantum programs
+operating on a large number of qubits, called **compiled
+mode**. Compiled mode can be enabled by adding ``-c`` to the command
+line options. Observe the speed-up in the benchmark:
+
+.. code::
```

Comment:
I also can't see this code section when I build.
<img width="672" alt="Screen Shot 2019-05-01 at 17 43 19" src="https://user-images.githubusercontent.com/44273795/57051896-af81d280-6c38-11e9-80ce-9d92475fef96.png">

Reply:
see above